chatbot/ai-studio.yaml

```diff
@@ -0,0 +1,27 @@
+application:
+  type: language
+  name: chatbot
+  description: This is a LLM chatbot application that can interact with a llamacpp model-service
+  containers:
+    - name: chatbot-inference-app
+      contextdir: ai_applications
+      containerfile: builds/Containerfile
+    - name: chatbot-model-service
+      contextdir: model_services
+      containerfile: base/Containerfile
+      model-service: true
+      backend: 
+        - llama
+      arch:
+        - arm64
+        - amd64
+    - name: chatbot-model-servicecuda
```

Comment:
seems to me there is an issue as if I run on amd64 and cuda I will get 2 containers being elected or we should revert the order and consider the name of the container is the same to filter

Reply:
filtering by name of container won't scale - users will be providing podman-desktop their own ai yamls, filtering by arch seems better to me

Reply:
filter by arch && gpu?

Reply:
closing as i think it is resolved after the discussion we had with Jeff. 
We're going to filter with arch && gpu